pkg/providers/aws/provider_postgres.go

```diff
@@ -343,6 +343,13 @@ func (p *PostgresProvider) reconcileRDSInstance(ctx context.Context, cr *v1alpha
 			Database: *foundInstance.DBName,
 			Port:     int(*foundInstance.Endpoint.Port),
 		}
+		// set the rds engine version in the status
+		if cr.Status.VersionDetails != *foundInstance.EngineVersion {
+			cr.Status.VersionDetails = *foundInstance.EngineVersion
+			if err := p.Client.Update(ctx, cr); err != nil {
```

Comment:
I think we can just return `nil, message, err` here and the calling function will do a the p.Client.Update.

Reply:
yea wasn't to sure the in memory cr wouldn't get over written at some point after the return. worth trying it out to see .

Reply:
after trying I don't need to return here just need to let it run on to the final return of the function